test/xml/CatLibrary/Tom.xml

```diff
@@ -50,6 +68,21 @@
       </Docs>
     </Member>
     <Member MemberName="TomMethod">
+      <Metadata>
+        <Meta Name="xamlMemberSyntax" Value="&lt;Tom method=&quot;TomMethod(...)&quot;/&gt;" />
+        <Meta Name="requirement_sdk_names" Value="Cat SDK" />
```

Comment:
do we really need to specify the same values on member level if it's already set on type level?